src/main/java/yevhen/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package yevhen.bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig (
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+)
+public class MapperConfig {
```

Comment:
You don't need MapperConfig annotation, just use @Mapper and implementationPackage unnecessary because it uses by default <PACKAGE_NAME>